servicecontrol/masstransit/docker-deployment.md

```diff
@@ -0,0 +1,84 @@
+---
+title: MassTransit Connector Docker Deployment
+summary: Configurating and running the MassTransit connector and the rest of the Particular Platform
+reviewed: 2024-12-02
+component: ServiceControl
+---
+
+## Overview
+
+When the MassTransit is used with the rest of the platfrom there are in total 4 containers that get deployed:
+
+1. ServiceControl
+1. ServiceControl-RavenDB
+1. MassTransit-Connector
+1. ServicePulse
+
+The ServiceControl and ServiceControl-RavenDB containers are used to ingest messages from the `error` queue. The MassTransit-Connector container reads messages from the MassTransit system's `_error` queues and moves them into the `error` queue to be ingested by ServiceControl. The ServicePulse container exposes a web interface to allow users to retry failed messages back to the MassTransit system.
+
+## Docker Compose 
```

Comment:
This section needs to be updated

Reply:
yes, I will delete the sampla and remove these